common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryContext.java

```diff
@@ -79,9 +80,7 @@ void addApplicationInfo(@NonNull final Context context) {
             put(App.NAME, packageInfo.applicationInfo.packageName);
             put(App.VERSION, packageInfo.versionName);
 
-            // Suppressing deprecation warnings due to deprecated property versionCode. There is already an existing issue for this: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/863
-            @SuppressWarnings(WarningType.deprecation_warning)
-            int versionCode = packageInfo.versionCode;
+            long versionCode = PackageInfoCompat.getLongVersionCode(packageInfo);
```

Comment:
From reading the docs, it looks like the format of this value has changed, but that this is _by design_ with the deprecation of this older API.

Thanks @PaulMuriithi!